packages/design-system/src/ToggleButton/ToggleButton.mdx

```diff
@@ -44,4 +44,18 @@ Constraint: Our toggle button currently only displays icons. It cannot display t
 - The action does not involve toggling or switching states.
 - Additional customization or functionality is required.
 
+**Size**
```

Comment:
Please use the markdown conventions for heading and stories that are in use by the rest of the documentation pages